spring-integration-jdbc/src/main/sql/destroy.sql.vpp

```diff
@@ -1,5 +1,9 @@
-DROP TABLE $!{IFEXISTSBEFORE} INT_MESSAGE $!{IFEXISTS};
-DROP TABLE $!{IFEXISTSBEFORE} INT_MESSAGE_GROUP $!{IFEXISTS};
-DROP TABLE $!{IFEXISTSBEFORE} INT_GROUP_TO_MESSAGE $!{IFEXISTS};
-DROP TABLE $!{IFEXISTSBEFORE} INT_LOCK $!{IFEXISTS};
-DROP INDEX $!{IFEXISTSBEFORE} INT_MESSAGE_IX1 $!{IFEXISTS};
+DROP INDEX$!{IFEXISTSBEFORE}INT_MESSAGE_IX1$!{IFEXISTS};
```

Comment:
Space missing before `$!{IFEXISTSBEFORE}` causes invalid syntax in generated scripts.